### PR TITLE
Enhancement for the uninstall_openshift_logging

### PR DESCRIPTION
**Pull request by Shrivaibavi**
Signed-off-by: Shrivaibavi Raghaventhiran <sraghave@redhat.com>

### CODE REVIEW

File: ocs_ci/utility/uninstall_openshift_logging.py [hunk 1]
```diff
@@ -50,6 +51,22 @@ def uninstall_cluster_logging():
...
+
+    # Deleting the RBAC permission set
+    rbac_role = ocp.OCP(
+        kind=constants.ROLE, namespace='openshift-operators-redhat'
+    )
+    rbac_role.delete(yaml_file=constants.EO_RBAC_YAML)
+
     # Deleting the projects
     openshift_logging_namespace = ocp.OCP(
         kind=constants.NAMESPACES, namespace='openshift-logging'
```
Comment (shreekarSS) [inline ocs_ci/utility/uninstall_openshift_logging.py]: same with   'openshift-logging'

File: ocs_ci/utility/uninstall_openshift_logging.py [hunk 2]
```diff
@@ -50,6 +51,22 @@ def uninstall_cluster_logging():
...
+        kind=constants.PVC, namespace='openshift-logging'
+    )
+    pvc_list = get_all_pvcs(namespace='openshift-logging')
+    for pvc in range(len(pvc_list) - 1):
+        pvc_obj.delete(resource_name=pvc_list['items'][pvc]['metadata']['name'])
+        pvc_obj.wait_for_delete(resource_name=pvc_list['items'][pvc]['metadata']['name'])
+
+    # Deleting the RBAC permission set
+    rbac_role = ocp.OCP(
+        kind=constants.ROLE, namespace='openshift-operators-redhat'
```
[the remark this answers is not in the document]
Comment (Shrivaibavi) [reply ocs_ci/utility/uninstall_openshift_logging.py]: done

File: ocs_ci/utility/uninstall_openshift_logging.py (hunk 1 above)
Comment (Shrivaibavi) [reply ocs_ci/utility/uninstall_openshift_logging.py]: done

**Comment by Shrivaibavi** [comment]
```The kubeconfig file /home/sraghave/test-ocs-sep-3/auth/kubeconfig doesn't exist!
Collecting Cluster versions
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.7.0, pytest-5.1.2, py-1.8.0, pluggy-0.12.0
rootdir: /home/sraghave/automation_ocs4/new_vai/ocs-ci, inifile: pytest.ini
plugins: logger-0.5.1, reportportal-1.0.5, metadata-1.8.0, html-1.22.0, marker-bugzilla-0.9.1.dev2
collected 1 item                                                                                                                                                                                                  

tests/test_uninstall.py::test_fun 
------------------------------------------------------------------------------------------------- live log setup --------------------------------------------------------------------------------------------------
15:12:36 - MainThread - tests.conftest - INFO - All logs located at /tmp/ocs-ci-logs-1568626937
15:12:36 - MainThread - ocs_ci.deployment.factory - INFO - Deployment key = aws_ipi
15:12:36 - MainThread - ocs_ci.deployment.factory - INFO - Current deployment platform: AWS,deployment type: ipi
15:12:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: ./bin/oc version
15:12:39 - MainThread - ocs_ci.utility.utils - INFO - OpenShift Client version: Client Version: version.Info{Major:"", Minor:"", GitVersion:"v4.2.0-alpha.0-2-g8fdb79e5", GitCommit:"8fdb79e549651c0f3c91d54349715309b5d149d3", GitTreeState:"clean", BuildDate:"2019-08-05T22:00:15Z", GoVersion:"go1.12.6", Compiler:"gc", Platform:"linux/amd64"}
                                                      Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.6+ee31d0b", GitCommit:"ee31d0b", GitTreeState:"clean", BuildDate:"2019-09-14T04:57:39Z", GoVersion:"go1.12.8", Compiler:"gc", Platform:"linux/amd64"}
                                                      OpenShift Version: 4.2.0-0.nightly-2019-09-15-221449
15:12:39 - MainThread - ocs_ci.utility.utils - INFO - The rook directory /home/sraghave/automation_ocs4/installation/ocs-ci/data/rook already exists, ocs-ci will skip the clone of rook repository.
15:12:39 - MainThread - ocs_ci.utility.utils - INFO - Fetching latest changes from rook repository.
15:12:39 - MainThread - ocs_ci.utility.utils - INFO - Executing command: git fetch --all
15:13:11 - MainThread - ocs_ci.utility.utils - INFO - Checkout rook repository to specific branch: master
15:13:11 - MainThread - ocs_ci.utility.utils - INFO - Executing command: git checkout master
15:13:11 - MainThread - ocs_ci.utility.utils - WARNING - Command warning: Already on 'master'
15:13:11 - MainThread - ocs_ci.utility.utils - INFO - Reset branch: master with latet changes
15:13:11 - MainThread - ocs_ci.utility.utils - INFO - Executing command: git reset --hard origin/master
-------------------------------------------------------------------------------------------------- live log call --------------------------------------------------------------------------------------------------
15:13:11 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get Pod  -n openshift-logging -o yaml
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace cluster-logging-operator-7f5c9596c4-b9gm9
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace elasticsearch-cdm-n2r502fs-1-6868b89cdf-z2fzk
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace elasticsearch-cdm-n2r502fs-2-5dfc497f7-rt4k4
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace elasticsearch-cdm-n2r502fs-3-6d6f7d4dbb-6gspm
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace fluentd-7bl6z
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace fluentd-jw9pm
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace fluentd-kjsqz
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace fluentd-m49ql
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace fluentd-mfb49
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace fluentd-mgtt7
15:13:13 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - Pods running in the openshift-logging namespace kibana-599dcd7cff-6n8pz
15:13:13 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig delete ClusterLogging instance
15:13:15 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get Pod  -n openshift-logging -o yaml
15:13:17 - MainThread - ocs_ci.utility.retry - WARNING - , Retrying in 30 seconds...
15:13:47 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get Pod  -n openshift-logging -o yaml
15:13:49 - MainThread - ocs_ci.utility.retry - WARNING - , Retrying in 60 seconds...
15:14:49 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get Pod  -n openshift-logging -o yaml
15:14:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get PersistentVolumeClaim  -n openshift-logging -o yaml
15:14:51 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig delete PersistentVolumeClaim elasticsearch-elasticsearch-cdm-n2r502fs-1
15:14:52 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get PersistentVolumeClaim elasticsearch-elasticsearch-cdm-n2r502fs-1 -n openshift-logging -o yaml
15:14:54 - MainThread - ocs_ci.ocs.ocp - INFO - PersistentVolumeClaim elasticsearch-elasticsearch-cdm-n2r502fs-1 got deleted successfully
15:14:54 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig delete PersistentVolumeClaim elasticsearch-elasticsearch-cdm-n2r502fs-2
15:14:55 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get PersistentVolumeClaim elasticsearch-elasticsearch-cdm-n2r502fs-2 -n openshift-logging -o yaml
15:14:57 - MainThread - ocs_ci.ocs.ocp - INFO - PersistentVolumeClaim elasticsearch-elasticsearch-cdm-n2r502fs-2 got deleted successfully
15:14:57 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig delete PersistentVolumeClaim elasticsearch-elasticsearch-cdm-n2r502fs-3
15:14:58 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get PersistentVolumeClaim elasticsearch-elasticsearch-cdm-n2r502fs-3 -n openshift-logging -o yaml
15:14:59 - MainThread - ocs_ci.ocs.ocp - INFO - PersistentVolumeClaim elasticsearch-elasticsearch-cdm-n2r502fs-3 got deleted successfully
15:14:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-operators-redhat --kubeconfig /home/sraghave/Downloads/auth/kubeconfig delete -f /home/sraghave/automation_ocs4/installation/ocs-ci/ocs_ci/templates/openshift-infra/logging-deployment/elasticsearch_operator/eo-rbac.yaml
15:15:01 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get Namespaces  -n openshift-logging -o yaml
15:15:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig delete Namespaces openshift-logging
15:15:27 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-logging --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get Namespaces openshift-logging -n openshift-logging -o yaml
15:15:29 - MainThread - ocs_ci.ocs.ocp - INFO - Namespaces openshift-logging got deleted successfully
15:15:29 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - The namespace openshift-logging got deleted successfully
15:15:29 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-operators-redhat --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get Namespaces  -n openshift-operators-redhat -o yaml
15:15:30 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-operators-redhat --kubeconfig /home/sraghave/Downloads/auth/kubeconfig delete Namespaces openshift-operators-redhat
15:15:56 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-operators-redhat --kubeconfig /home/sraghave/Downloads/auth/kubeconfig get Namespaces openshift-operators-redhat -n openshift-operators-redhat -o yaml
15:15:58 - MainThread - ocs_ci.ocs.ocp - INFO - Namespaces openshift-operators-redhat got deleted successfully
15:15:58 - MainThread - ocs_ci.utility.uninstall_openshift_logging - INFO - The project openshift-opertors-redhat got deleted successfully
PASSED                                                                                                                                                                                                      [100%]

========================================================================================== 1 passed in 201.25s (0:03:21) ==========================================================================================```

**Pull request merged**